### PR TITLE
feat(common):first commit

### PR DESCRIPTION
**Pull request by Zwilson135**
first commit of my project

### CODE REVIEW

File: src/Theme/ThemeListP.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        letterSpacing:'0.44px',
+        textAlign:'left',
+        fontStretch:'normal',
+        textDecoration:'none',
+      }
+    },
+
+  })
+export default ThemeListP
+
```
Comment (Ditmar) [inline src/Theme/ThemeListP.jsx]: please put this code in the theme main file, only one theme can exist for the whole app. put this here 
https://github.com/DanX3N0/frontend-rojo/blob/main/src/Theme/Theme.js

File: src/ui/components/ListaProductos/ListProducts.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Header from './header/Header'
+import SideNavbar from './sideNavbar/SideNavbar'
+import Table from "./table/Table"
+import ThemeListP from '../../../Theme/ThemeListP'
+import { Container,CssBaseline,ThemeProvider} from "@mui/material"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/ListProducts.jsx]: please use single quotes, and use and correct the indentation

File: src/ui/components/ListaProductos/ListProducts.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import FilterComponent from "./filter/FilterComponent"
+
+const ListProducts = () => {
+  return (
+    <ThemeProvider theme={ThemeListP}>
+    <CssBaseline/>
+      <Container sx={{width:'1440px',height:'1024px',background:'#E6E6E8'}}>
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/ListProducts.jsx]: Don't hard code the colors use the main palette from the theme file like this. 
```
import { useTheme } from '@material-ui/core'
.
.
const theme = useTheme();
.
.
<Container sx={{width:'1440px',height:'1024px',background:theme.palette.primary.somecolor}}>
>
```

File: src/ui/components/ListaProductos/filter/BpCheckedIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Checkbox, { CheckboxProps } from '@mui/material/Checkbox';
+
+const BpIcon = styled('span')(({ theme }) => ({
+  borderRadius: 6,
+  width: 18,
+  height: 18,
+  border: '2px solid #CED4DA',
+  backgroundColor: theme.palette.primary.main,
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/BpCheckedIcon.tsx]: Nice catch, this is the way to include the color, feel free to add the colors if you need inside the color palette.

File: src/ui/components/ListaProductos/filter/BpCheckedIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      disableRipple
+      color="default"
+      checkedIcon={<BpCheckedIcon />}
+      icon={<BpIcon />}
+      inputProps={{ 'aria-label': 'Checkbox' }}
+      {...props}
+    />
+  );
+}
+*/
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/BpCheckedIcon.tsx]: remove all commented code

File: src/ui/components/ListaProductos/filter/DividerGroup.jsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    width:'194px',
+    height:'1px',
+    zIndex:1,
+    position:'absolute',
+    left:'19px',
+    backgroundColor:theme.palette.filter.divider,
+    
+  }));
+
+function DividerGroup(props) {
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/DividerGroup.jsx]: please use the definition of a function like this
const dividerGroup = () => {
}

File: src/ui/components/ListaProductos/filter/FilterComponent.jsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {Typography,Stack,FormLabel} from '@mui/material'
+import {StyledContainerFilt,StylesLayoutTitle,StyledTitleFilt,StyledGroupFilt} from "./stylesFilter"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/FilterComponent.jsx]: correct the indentation and use simple quotes

File: src/ui/components/ListaProductos/filter/FormControlCategory.jsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { FormControl,FormLabel,FormGroup,FormControlLabel,Typography,Divider} from '@mui/material'
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/FormControlCategory.jsx]: correct the indentation

File: src/ui/components/ListaProductos/filter/FormControlCategory.jsx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+export const FormControlCategory = () => {
+  return (
+    <FormControl component="fieldset" variant="standard">
+        <FormGroup sx={StylesFormGroupCategory}>
+        <FormLabel><CustomTypographySub subtitle={"Categoria"}/></FormLabel>
+          <FormControlLabel control={<BpCheckbox name="mujer" checked/>} label={
+              <Typography  sx={{...DefaultTypographyItem,color:'secondary.main',width:'38px'}}>
+                Mujer
+              </Typography>} 
+          sx={{...defaultStyleFormItem,width:'66px'}}
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/filter/FormControlCategory.jsx]: the better way to set up the breakpoint is using our function mq from the css 
https://github.com/DanX3N0/frontend-rojo/blob/main/src/config/mq.js

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import AddCard from "./AddCard"
+import SearchBar from "./SearchBar";
+import UserAvatar from "./UserAvatar"
+import AvatarImage from "../../../../assets/images/avatarImage.jpg"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 11]
```diff
@@ -0,0 +1,25 @@
+import AddCard from "./AddCard"
+import SearchBar from "./SearchBar";
+import UserAvatar from "./UserAvatar"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 12]
```diff
@@ -0,0 +1,25 @@
+import AddCard from "./AddCard"
+import SearchBar from "./SearchBar";
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 13]
```diff
@@ -0,0 +1,25 @@
+import AddCard from "./AddCard"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+import SearchBar from "./SearchBar";
+import UserAvatar from "./UserAvatar"
+import AvatarImage from "../../../../assets/images/avatarImage.jpg"
+import {Toolbar,Box} from "@mui/material"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/header/Header.jsx [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+import { StyledBarHeader,StylesToolbar} from "./styles"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/header/Header.jsx]: use simple quotes here

File: src/ui/components/ListaProductos/icons/HomeIcon.jsx [hunk 16]
```diff
@@ -0,0 +1,13 @@
+import { Icon } from '@mui/material';
+
+function HomeIcon(props) {
+  return (
+    <Icon {...props}>
+        <svg width="20px" height="20px">
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/HomeIcon.jsx]: don't copy all svg code in html plain text. you can export the file into the assets and use it directly, like this.you can use 

```
import React from 'react';
import SvgIcon from '@material-ui/core/SvgIcon';
import { ReactComponent as ImportedSvg } from '../assets/logo.svg';

const MyComponent = () => {
  return (
    <SvgIcon component={ImportedSvg} viewBox="0 0 24 24" />
  );
};

export default MyComponent;
```

File: src/ui/components/ListaProductos/icons/OpinionsIcon.jsx [hunk 17]
```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@mui/material';
+
+function OpinionsIcon(props) {
+  return (
+    <Icon {...props}>
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/OpinionsIcon.jsx]: read the comment above

File: src/ui/components/ListaProductos/icons/OrdersIcon.jsx [hunk 18]
```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@mui/material';
+
+function OrdersIcon(props) {
+  return (
+    <Icon {...props}>
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/OrdersIcon.jsx]: read the comment above

File: src/ui/components/ListaProductos/icons/PromotionsIcon.jsx [hunk 19]
```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@mui/material';
+
+function PromotionsIcon(props) {
+  return (
+    <Icon {...props}>
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/PromotionsIcon.jsx]: read the comment above

File: src/ui/components/ListaProductos/icons/SearchIcon.jsx [hunk 20]
```diff
@@ -0,0 +1,11 @@
+import SvgIcon from '@mui/material/SvgIcon';
+
+function SearchIcon(props) {
+  return (
+    <SvgIcon {...props} viewBox="0 0 15 15">
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/SearchIcon.jsx]: read the comment above

File: src/ui/components/ListaProductos/icons/SettingIcon.jsx [hunk 21]
```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@mui/material';
+
+function PromotionsIcon(props) {
+  return (
+    <Icon {...props}>
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/SettingIcon.jsx]: read the comment above

File: src/ui/components/ListaProductos/icons/SettingIcon.jsx [hunk 22]
```diff
@@ -0,0 +1,12 @@
+import { Icon } from '@mui/material';
+
+function PromotionsIcon(props) {
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/icons/SettingIcon.jsx]: use the sentence like this const PromotionsIcon = props => {
}

File: src/ui/components/ListaProductos/logo/styles.jsx [hunk 23]
```diff
@@ -0,0 +1,12 @@
+import {styled} from "@mui/system"
```
Comment (Ditmar) [inline src/ui/components/ListaProductos/logo/styles.jsx]: use simple quotes

**Review by Ditmar — CHANGES_REQUESTED** [review]
This pr is so big, please review your scope

**Comment by Zwilson135** [comment]
ingeniero ya le hice las correcciones como dijo

**Comment by olmedo90** [comment]
image

It is not according to Figma. correct the following: (image of the products, title of the selected checkboxes)

**Comment by olmedo90** [comment]
image

It is according to Figma. continue with the same interest. 👍

**Review by Ditmar — APPROVED** [review]
approved, please rebase all your code to proceded  to merge into the master branch

**Pull request merged**